projects/ad4630_fmc/README.md

```diff
@@ -9,12 +12,12 @@
 
 | Part name                                                    | Description                                                  |
 |--------------------------------------------------------------|--------------------------------------------------------------|
-| [AD4030](https://www.analog.com/en/products/ad4030-24)       | 24-Bit, 2 MSPS, Single Channel SAR ADC                       |
+| [AD4030](https://www.analog.com/ad4030-24)                   | 24-Bit, 2 MSPS, Single Channel SAR ADC                       |
 | [AD4630](https://www.analog.com/ad4630-24)                   | 24-Bit, 2 MSPS, Dual Channel, Precision Differential SAR ADC |
-| [AD4632](https://www.analog.com/AD4632-24)                   | 24-Bit, 500 kSPS, Dual Channel SAR ADC                       |
-| [ADAQ4216](https://www.analog.com/en/products/adaq4216.html) | 16-Bit, 2MSPS, μModule Data-Acquisition Solution             |
-| [ADAQ4224](https://www.analog.com/en/products/adaq4224.html) | 24-Bit, 2 MSPS, μModule Data Acquisition Solution            |
+| [AD4632](https://www.analog.com/ad4632-24)                   | 24-Bit, 500 kSPS, Dual Channel SAR ADC                       |
+| [ADAQ4216](https://www.analog.com/adaq4216)                  | 16-Bit, 2 MSPS, μModule Data-Acquisition Solution            |
+| [ADAQ4224](https://www.analog.com/adaq4224)                  | 24-Bit, 2 MSPS, μModule Data Acquisition Solution            |
 
 ## Building the project
 
-Please enter the folder for the FPGA carrier you want to use and read the README.md.
\ No newline at end of file
+Please enter the folder for the FPGA carrier that you want to use and read the README.md.
```

Comment:
Please revert this change

Reply:
why?

Reply:
Because it is not according to the template, and that phrase is already grammatically correct as it is